lib/DataBase/bd_itrek.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:sqflite/sqflite.dart';
+import 'package:path/path.dart';
+
+class DatabaseHelper {
+  static final DatabaseHelper instance = DatabaseHelper._init();
+  static Database? _database;
+
+  DatabaseHelper._init();
+
+  Future<Database> get database async {
+    if (_database != null) return _database!;
+    _database = await _initDB('itrek_database.db');
+    return _database!;
+  }
+
+  Future<Database> _initDB(String filePath) async {
+    final dbPath = await getDatabasesPath();
+    final path = join(dbPath, filePath);
+
+    return await openDatabase(path, version: 1, onCreate: _createDB);
+  }
+
+  Future _createDB(Database db, int version) async {
+    await db.execute('''
+CREATE TABLE IF NOT EXISTS users (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  username TEXT NOT NULL,
+  password TEXT NOT NULL,
+  token TEXT
+)
```

Comment:
Sería mejor una tabla para valores, que no solo sirva para usuarios (que solo tendremos uno por teléfono)
```suggestion
CREATE TABLE IF NOT EXISTS variables (
  key TEXT PRIMARY KEY,
  value TEXT
);
```

luego se usa así
`SELECT value FROM variables WHERE key = 'auth_token';`
`DELETE FROM variables WHERE key = 'auth_token';`

Reply:
en ves de user , se lo cambio a valores o parámetros. ok